asreview/review/simulate.py

```diff
@@ -32,23 +32,24 @@ def __init__(self,
         super(ReviewSimulate, self).__init__(
             X, y, *args, **kwargs)
 
-    def _prior_knowledge(self):
+    def _prior_knowledge(self, logger):
```

Comment:
Can you clarify this one. `logger` doesn't do anything here. Why can't `classify()` access the logger?

Reply:
There is no logger in the Review class anymore, just the filename. We don't want to open/close a file every time we classify a sample.

Reply:
So you pass an open logger to classify? Why not store the open logger in the object (self)?

Reply:
Because it could lead to a bad logger? I.e. you open the file, do reviews, close the file, then access the closed file. Of course you can safeguard against that, etc. But I think that would be about (or more) painful as the current solution?